portals/publisher/source/src/app/components/Apis/Details/NewOverview/Endpoints.jsx

```diff
@@ -26,7 +26,14 @@ import Box from '@material-ui/core/Box';
 import { withAPI } from 'AppComponents/Apis/Details/components/ApiContext';
 
 const showEndpoint = (api, type) => {
-    if (api.endpointConfig) {
+    if (api.advertiseInfo.advertised) {
```

Comment:
use a null check for advertiseInfo

Reply:
Added with https://github.com/wso2/apim-apps/pull/130/commits/e73f69c14b5994afe7b2aa31142b5d0c2dc566e9